packages/components-storybook/test/integration/Buttons.test.js

```diff
@@ -1,47 +1,85 @@
-        'button_inverse_icon_error',
-        'button_inverse_icon_white',
-        'button_inverse_icon_transparent',
-        'button_inverse_icon_disabled',
+        'button_basic_icon',
+        'button_basic_icon_transparent',
+        'button_basic_icon_disabled',
+        'button_basic_loading',
+        'button_basic_loading_transparent',
+        'button_basic_loading_disabled',
+        'button_full_width',
+        'button_full_width_transparent',
+        'button_full_width_disabled',
+        'button_full_width_right',
+        'button_full_width_right_transparent',
+        'button_full_width_right_disabled',
+        'button_full_width_left',
+        'button_full_width_left_transparent',
+        'button_full_width_left_disabled',
+        'button_full_width_icon',
+        'button_full_width_icon_transparent',
+        'button_full_width_icon_disabled',
+        'button_full_width_icon_right',
+        'button_full_width_icon_right_transparent',
+        'button_full_width_icon_right_disabled',
+        'button_full_width_icon_left',
+        'button_full_width_icon_left_transparent',
+        'button_full_width_icon_left_disabled',
+        'button_full_width_loading',
+        'button_full_width_loading_transparent',
+        'button_full_width_loading_disabled',
+        'button_full_width_loading_right',
+        'button_full_width_loading_right_transparent',
+        'button_full_width_loading_right_disabled',
+        'button_full_width_loading_left',
+        'button_full_width_loading_left_transparent',
+        'button_full_width_loading_left_disabled',
     ].forEach(testName => {
         it(`${testName}`, () => {
+            cy.loadContent('/iframe.html?selectedKind=Buttons&selectedStory=All&full=0');
+            if (testName.match(/icon/)) {
+                cy.getTestElement(testName)
+                    .find('.loading')
+                    .each(el => {
+                        cy.get(el).should('not.exist');
+                    });
+
+                cy.getTestElement(testName)
+                    .find('svg')
+                    .should('be.visible');
+            }
+
+            cy.getTestElement(testName)
+                .should('be.visible')
+                .matchImageSnapshot();
+        });
+    });
+
+    [
+        'button_inverse',
+        'button_inverse_icon',
+        'button_inverse_loading',
+
+        'button_inverse_full_width',
+        'button_inverse_icon_full_width',
+        'button_inverse_loading_full_width',
+
+        'button_inverse_full_width_right',
+        'button_inverse_icon_full_width_right',
+        'button_inverse_loading_full_width_right',
+
+        'button_inverse_full_width_left',
+        'button_inverse_icon_full_width_left',
+        'button_inverse_loading_full_width_left',
+    ].forEach(testName => {
+        it(`${testName}`, () => {
+            cy.loadContent('/iframe.html?selectedKind=Buttons&selectedStory=Inverse&full=0');
```

Comment:
Hierarchy inside storybook menu is a bit confusing
- Buttons
  - All
  - Button
  - Pin Button
  - Inverse

where on “All” page are not all buttons and its variants as one would expect, the rest of it is on a separate “Inverse” page. It is a bit confusing, it also looks like the Inverse page is for different component (just like Button and Pin Button are). Is this change necessary? I would stick to only using one page (All) for showing all the different props.

Reply:
I put everything under All section, deeleted inverse.tsx